src/cluecode/finder.py

```diff
@@ -74,6 +90,12 @@ def find(location, patterns):
 
     for i, line in enumerate(analysis.text_lines(location)):
         lineno = i + 1
```

Comment:
Since you are touching this code, I think that this could be rewritten as:
```
     for lineno, line in enumerate(analysis.text_lines(location), start=1):
```
e.g. incrementing a separate counter does not make sense IMHO